### ISSUE TITLE
How to extract common chunks in multiple resource files

### ISSUE THREAD

**Issue by yvanwangl** [skpm/skpm#249]
As the resource files are build by independent webpack compiler. So how can we extract common chunks in multiple resource files?

**Comment by mathieudutour** [comment]
That's a good question, and I don't know the answer hahaha

Any reason you want to do that? Reduce the plugin size?

**Comment by yvanwangl** [comment]
@mathieudutour Yes, when I build my plugin, I found that third-party ui libraries are packaged in every webView js resources, so the plugin size is so huge. Now the size is larger than 33M.

When I read the builder code, I found the resources files are mapped to be build by webpack:
```
for (const entry of entries) {
    const compiler = webpack(await webpackConfig(entry))
    if (watch) {
      // https://github.com/webpack/webpack.js.org/issues/125
      // watchOptions need to be manually passed to the watch() method.
      compilers.push(
        compiler.watch(
          compiler.options.watchOptions,
          buildCallback(entry.script, watch)
        )
      )
    } else {
      compiler.run(buildCallback(entry.script))
    }
  }

```
[source code](https://github.com/skpm/skpm/blob/master/packages/builder/src/build.js#L352)
If we pass the entries to webpack directly, we can use only one webpack compiler to build mutil-entries, in this way we can extract common chunks in multiple resource files.
How do you think about it? 
I look forward to hearing from you.
